package.json

```diff
@@ -50,30 +50,46 @@
     "@fortawesome/free-regular-svg-icons": "5.15.1",
     "@fortawesome/free-solid-svg-icons": "5.15.1",
     "@ionic/angular": "^7.0.0",
+    "@ionic/cli": "~7.2.0",
     "@ionic/cordova-builders": "^10.0.0",
     "@ionic/storage-angular": "4.0.0",
     "@mobiscroll/angular": "npm:@mobiscroll/angular-ivy-trial@^5.27.3",
     "@mobiscroll/cli": "^1.14.1",
-    "@types/geojson": "^7946.0.12",
+    "@types/geojson": "^7946.0.14",
     "@types/node": "^12.11.1",
     "@ungap/global-this": "^0.4.4",
     "angular2-uuid": "1.1.1",
+    "assert": "^2.1.0",
+    "cordova": "~12.0.0",
     "cordova-android-support-gradle-release": "^3.0.1",
-    "cordova-plugin-advanced-http": "^3.1.0",
+    "cordova-plugin-advanced-http": "^3.3.1",
+    "cordova-plugin-androidx": "^3.0.0",
     "cordova-plugin-androidx-adapter": "^1.1.3",
-    "cordova-plugin-camera": "^4.0.3",
-    "cordova-plugin-device": "2.0.2",
-    "cordova-plugin-ionic-keyboard": "^2.0.5",
-    "cordova-plugin-ionic-webview": "^5.0.0",
-    "cordova-plugin-splashscreen": "5.0.2",
-    "cordova-plugin-statusbar": "2.4.2",
+    "cordova-plugin-camera": "^4.1.0",
+    "cordova-plugin-device": "^2.0.2",
+    "cordova-plugin-file": "^8.0.1",
+    "cordova-plugin-geolocation": "^4.1.0",
+    "cordova-plugin-googlemaps": "^2.7.1",
```

Comment:
I think this plugin broke the iOS build - Might need to check if this builds on XCode first